scripts.js

```diff
@@ -0,0 +1,74 @@
+// Creates the canvas grid
+
+for (var i = 0; i < 1500; i++) {
+  console.log(i);
+  $('.canvas').append("<div class='grid'></div>");
+}
+
+// Selects and applies brush colors
+
+let currentColor = '#000000';
+
+$('.canvas').on('click', 'div', function () {
+  $(this).css('background-color', currentColor);
+});
+
+$('.blackButton').on('click', function () {
```

Comment:
all these click handlers for your palate are nearly identical. I would have put all the color hex values in an array and cycle through that array when you build out the palate. Attach them through a data attr. Then just create one click handler attached to the palate as a whole that grabs the "color" data attr of the target item and uses that to set your bush color.